### PR TITLE
Setting up rails project

### LINKED ISSUE #1: Setup
# REQUIREMENTS
This issue is to generate the Ruby on Rails code base, which will allow to develop the project.

## Rails Version:  6.0.3.4

# Acceptance Criteria
Preferably use the command:

- [x] rails new tribo --database=postgresql --skip-sprockets
- [x] Rubocop setup to filter styling of our code

### PR DESCRIPTION
**Pull request by cardotrejos**
Add fresh rails project and some configs:

-.gitignore
-Rubocop
-Rails

### CODE REVIEW

**Review by edgarjaymez — CHANGES_REQUESTED** [review]
Change `database.yml` to `database.yml.example` and add the first one to `.gitignore` to make it easy for everyone to keep their DB Server credentials configured.

**Review by edgarjaymez — DISMISSED** [review]
Everything looks great!

**Comment by cardotrejos** [comment]
@YilKanda It's my first time setting up Rubocop, I add gems to development group and test group, also a .rubocop.yml defining some styles for rubocop. Would be nice to get feedback for this task.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+group :development do
+  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'rubocop'
```
Comment (YilKanda) [inline Gemfile]: When doing the style guide check with Rubocop, I got this error:

imagen

Add this line to the application's Gemfile:

`gem 'rubocop-packaging'`

It should be something like that in the Gemfile:

imagen

Comment (YilKanda) [reply Gemfile]: Using Rubocop allows us to identify style errors in the code:

For example, in this project, they already come by default:

imagen

And using Rubocop, they decrease to:

imagen

To execute it, just place in the console, the command:

`rubocop -A
`

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by YilKanda — DISMISSED** [review]
Looks great to me!

**Review by edgarjaymez — DISMISSED** [review]
Let's go!

**Review by jaimelr — COMMENTED** [review]
LGTM overall but let's try to start with a clean commit history.

My suggestion is to squash the commits where you apply the changes into the first commit. You can do an interactive rebase with `git rebase -i COMMIT-HASH`, let me know if you agree with this and if you need help performing the rebase.

For example: [This commit](https://github.com/BrightCoders-Proyectos/tribo-web/pull/11/commits/203b7a504aaa894d77330bbe86f75cb488e7ff68) says "adding requested changes" but changes can be squashed into the first commit since it is not adding new features but fixing issues introduced in the first commit. Unless you want to explicitly note that you made a fix there's no need to create another commit.

**Review by jaimelr — APPROVED** [review]
LGTM! 👍

**Review by edgarjaymez — APPROVED** [review]
👍

**Review by YilKanda — APPROVED** [review]

**Review by EduJimenez96 — APPROVED** [review]

**Review by EduJimenez96 — COMMENTED** [review]
👍

**Pull request merged**